Common/UnitDefinitions/RatioChangeRate.json

```diff
@@ -0,0 +1,32 @@
+﻿{
+  "Name": "RatioChangeRate",
+  "BaseUnit": "PercentPerSecond",
```

Comment:
I would choose `DecimalFractionPerSecond` as base unit, to be consistent with [Ratio](https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/Ratio.json).

You would then have to swap some of the conversion functions and test case values too.